pre-commit/web_lint_check.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+import multiprocessing
+import sys
+
+
+def web_lint_check():
+    if len(sys.argv) <= 2:
+        print("At least 2 arguments are required")
+        print("usage: python web_lint_check.py [js | css | misc] [<file_paths>]")
+        exit(2)
+
+    dir = "webui/react/"
+    target: str = sys.argv[1]
+    file_paths: str = " ".join(
+        [file_path.replace(dir, "") for file_path in sys.argv[2:]]
```

Comment:
This might be a little nicer with relpath, since that handles differnet OS path separators properly and is aware of special things like `../` and so on.
https://docs.python.org/3/library/os.path.html#os.path.relpath

```suggestion
        [os.path.relpath(file_path, dir) for file_path in sys.argv[2:]]
```

Reply:
nice catch! I was planning to use it but my hand get used to type `replace` :)